newrelic/hooks/framework_aiohttp.py

```diff
@@ -279,16 +263,13 @@ async def _coro():
             response = await wrapped(*args, **kwargs)
 
             try:
-                trace.process_response_headers(response.headers.items())
+                trace.process_response(status_code=response.status)
             except:
                 pass
 
             return response
-        except Exception as e:
-            try:
-                trace.process_response_headers(e.headers.items())
-            except:
-                pass
+        except Exception:
+            notice_error()
 
             raise
```

Comment:
I don't think we want to call `notice_error` here necessarily, it's not equivalent to what we had before. I think this outer try/except can be deleted and we end up with just the success case.

```
response = await wrapped(*args, **kwargs)

try:
    trace.process_response(status_code=response.status)
except:
    pass

return response
```

Reply:
I don't see any issues with that commit